scubagoggles/rego/Commoncontrols.rego

```diff
@@ -599,33 +664,58 @@ NonCompliantOUs4_1 contains {
     not IsGoodLimit(LastEvent.NewValue)
 }
 
-tests contains {
-    "PolicyId": "GWS.COMMONCONTROLS.4.1v0.3",
+# The following checks for web session duration less than or equal to 12 hours.
+# Not sure whether this really fits with 4.1 requirement.  The 12 hours is
+# mentioned, but the requirement deals with forced re-authentication after the
+# session expiration - is this automatic with session expiration?  If so, then
+# this check is probably appropriate.  It seems to fit with the event log
+# method above.
```

Comment:
Recommend trimming this comment - this is the setting relevant to 4.1, no uncertainty about it.
```suggestion
# The following checks for web session duration less than or equal to 12 hours.
```